### DISCUSSION TITLE
bib文件中未被引用的文献出现在参考文献列表中

### DISCUSSION THREAD

**Discussion by lyh458** [sikouhjw/gdutthesis#114] [Show and tell]
- 问题：部分未被引用的文献出现在参考文献列表中
- 解决办法：注释掉`main.tex`文件中的
```latex
\nocite{*}% 列出全部参考文献
```
建议维护者添加相关说明到user manual，毕竟.bib文件中放多几个参考文献应该是很常见的吧？
FYI.

**Comment by sikouhjw** [comment]
`main.tex` 内对应命令有中文注释，且手册只放“模板独有的功能”，而 `\nocite{*}` 是 latex 通用功能。

**Reply by lyh458** [reply]
说实话，原来的注释模棱两可。。。
```latex
\nocite{*}% 列出全部参考文献
```
`列出全部参考文献`会让人误以为是：列出所有引用的文献。
如果我没检查文献的数量和条目的话，肯定不会在意这个问题，更不会去修改这个选项。

**Reply by sikouhjw** [reply]
那我改成，过段时间提交
```tex
\nocite{*}% 列出 bib 文件中的全部参考文献
```

**Reply by muzimuzhi** [reply]
"列出全部参考文献，包括未引用的"
或者，默认注释 `\nocite{*}`。我不确定哪种更规范，但参考文献包含未被引用的文献，有点奇怪。

不过我同意，模板没有教用户 latex 基础使用（如，Lamport 书里提到的所有用法）的义务。

**Comment by sikouhjw** [comment]
> 那我改成，过段时间提交
> 
> ```
> \nocite{*}% 列出 bib 文件中的全部参考文献
> ```

https://github.com/sikouhjw/gdutthesis/commit/13d42a9f327ae296b899f5bfe48d0ce35f07cf2d 已提交
